.eslintrc.yaml

```diff
@@ -15,12 +15,17 @@ rules:
   no-var: error
   curly: error
   no-floating-decimal: error
-  no-unused-vars: error
-  # Disabling jsdoc rules for the time being, but should be discussed as to whether or not this is
-  # something we should add in. For closure compatibility, we can investigate using something like
-  # https://github.com/angular/tsickle with .dts files.
+  no-unused-vars:
+    - error
```

Comment:
You could also turn this off and enable Closure's unused var checking instead, with `--jscomp_error unusedLocalVariables`. I don't know if there are any subtle differences between ours and ESLint's other than it not knowing about `@typedef`s